MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -1814,6 +1820,10 @@ private void maybeShowPopup(MouseEvent e) {
                         menuItem.addActionListener(this);
 
                         setAwardMenu.add(menuItem);
+
+                        if (setAwardMenu.getItemCount() > 30) {
```

Comment:
Should this be smaller? We've had requests by folks with small vertical resolutions elsewhere.

Reply:
I'll change it to the 20 items like the spouse one, assuming that value is ok.

Reply:
I'll double check on my 13" MacBook today.

Reply:
You still left the `if` test at 30.